docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2050,15 +2430,31 @@ Empty by default.
 
 **Example**
 
-``` xml
-<replica_group_name>backups</replica_group_name>
+```xml
+<replica_group_name>
+    backups
+</replica_group_name>
 ```
 
 Type: String
 
 Default: ""
 
-## max_open_files
+## remap_executable
+
+Setting to reallocate memory for machine code ("text") using huge pages.
+
+:::note
+This feature is highly experimental, and therefore set `false` by default.
```

Comment:
Generally speaking, I think we should standardize the way we specify defaults.

Could we add in l. 2456 something like "Default: `false`"?